profiles/common/modules/features/multisite_custom_error/multisite_custom_error.info

```diff
@@ -8,3 +8,4 @@ dependencies[] = features
 dependencies[] = customerror
 features[features_api][] = api:2
 files[] = tests/multisite_custom_error.extra.test
+configure = admin/config/system/customerror
```

Comment:
This path `admin/config/system/customerror` is not part of the `multisite_custom_error` module so it should not be mentioned in the info file. It is part of the CustomError module, which is a different module :)

See [customerror_menu()](http://cgit.drupalcode.org/customerror/tree/customerror.module#n148).